packages/client/package.json

```diff
@@ -83,8 +83,9 @@
     "jwt-simple": "^0.5.6",
     "level": "^8.0.0",
     "memory-level": "^1.0.0",
-    "multiaddr": "^10.0.1",
-    "peer-id": "^0.14.3",
+    "@multiformats/multiaddr": "^12.1.3",
+    "@libp2p/interface-peer-id": "2.0.2",
+    "@libp2p/peer-id-factory": "2.0.4",
```

Comment:
Maybe @acolytec3 can confirm, but we removed libp2p completely from the core build, so I would think these two dependencies shouldn't show up here at all? 🤔

Reply:
Depends on how we're using `peer-id`. That library got radically changed a while back so you might need that factory for even our limited usage.

Reply:
Mostly found in `client/libp2pBrowserBuild`